packages/recommend-core/src/types/InitialState.ts

```diff
@@ -0,0 +1,5 @@
+import { ProductRecord } from './ProductRecord';
+
+export type InitialState<TObject> = {
```

Comment:
See [Slack thread](https://algolia.slack.com/archives/C03QWAMAN6P/p1668617762173699?thread_ts=1668597136.031949&cid=C03QWAMAN6P). There's still ongoing discussions about this naming before moving forward.

Reply:
✅ : https://github.com/algolia/recommend/pull/109/commits/ab1e664d713cbde4416ec8d201e5f97d601e6460

Reply:
If the object structure is `{ status, recommendations }`, then `initialResults` is more correct than `initialRecommendations`.

Reply:
the structure only has recommendations:
```
export type InitialRecommendations<TObject> = {
  recommendations: Array<ProductRecord<TObject>>;
};
```

Reply:
Yes, it _has_ recommendations, it _is not_ recommendations.

Reply:
✅ : https://github.com/algolia/recommend/pull/109/commits/dd46e702b525bf33cefd88a3222e5214adbea3ba